### PR TITLE
Mes 1109 icon splash screen

### PR DESCRIPTION
**Pull request by ammarhaiderbjss**
## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop
- [x] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [x] PR link added to JIRA
- [x] Reviewers selected in Github
- [x] Any new reusable component/widget added to component library on Confluence

- [x] Screenshot(s) captured on the physical device (if applicable)
- [x] Tested by QA
- [x] PO's approval

Icon: 

img_c9d9a30b1c54-1

Splashscreen:

img_0fce3c44bc7b-1

### CODE REVIEW

File: config.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version='1.0' encoding='utf-8'?>
-<widget id="uk.gov.dvsa.mobile-examiner" version="1.0.0" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
+<widget id="uk.gov.dvsa.mobile-examiner" version="1.0.1543422786" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
```
Comment (johnmorleybjss) [inline config.xml]: Version bumped.

Comment (johnmorleybjss) [reply config.xml]: Perhaps we should look into only running the versioning hook when a flag is provided to the build command? (Providing this is something we could configure on Ionic Pro.

Comment (ammarhaiderbjss) [reply config.xml]: Thanks for spotting, I was meant to revert this back to what's on develop, will do it now

Comment (ammarhaiderbjss) [reply config.xml]: And yes, should only bump the version when we build on Ionic Pro

Comment (johnmorleybjss) [reply config.xml]: Okay, will have to look into what we can configure for this on Ionic Pro and update the build hook accordingly.

**Pull request merged**